provisions/roles/scanner/tasks/main.yml

```diff
@@ -42,24 +42,27 @@
     mode: u+x
   with_items:
     - ../beanstalk_worker/beanstalkc.py
-    - ../beanstalk_worker/worker_start_test.py
+    - ../beanstalk_worker/worker_start_scan.py
   when: not vagrant
 
 - name: Get service files for workers
   copy:
     src: "{{ item.src }}"
     dest: "{{ item.dest }}"
   with_items:
-      - {src: "../beanstalk_worker/cccp-test-worker.service", dest: /etc/systemd/system/cccp-test-worker.service}
+      - {src: "../beanstalk_worker/cccp-scan-worker.service", dest: /etc/systemd/system/cccp-scan-worker.service}
 
-- name: Skip atomic scan for test environment
+- name: Skip atomic scan for scan environment
   replace: >
-    dest=/etc/systemd/system/cccp-test-worker.service
-    regexp='worker_start_test.py$'
-    replace='worker_start_test.py ci'
-  when: test
+    dest=/etc/systemd/system/cccp-scan-worker.service
+    regexp='worker_start_scan.py$'
+    replace='worker_start_scan.py ci'
+  when: scan
```

Comment:
@rtnpro: Because it's inappropriate. We're **scanning** the container images but the worker for that is named as **test**. And like worker, most things around it are also named with **test**.